ydb/core/viewer/viewer_tenantinfo.h

```diff
@@ -718,17 +750,23 @@ class TJsonTenantInfo : public TViewerPipeClient {
                         }
                     }
 
-                    for (const auto& [type, size] : storageUsageByType) {
-                        auto& storageUsage = *tenant.AddStorageUsage();
-                        storageUsage.SetType(type);
-                        storageUsage.SetSize(size);
+                    for (const auto& [type, size] : tablesStorageByType) {
                         auto it = storageQuotasByType.find(type);
-                        if (it != storageQuotasByType.end()) {
-                            storageUsage.SetLimit(it->second.HardQuota);
-                            storageUsage.SetSoftQuota(it->second.SoftQuota);
-                            storageUsage.SetHardQuota(it->second.HardQuota);
+                        if (it != storageQuotasByType.end() && it->second.SoftQuota) {
```

Comment:
why do we need check for SoftQuota? we won't be able to see tablet storage size without quota?

Reply:
I reverted it to output as much information as possible. Based on the comments in the issue, I thought that the information wasn't needed without a quota.